api/v1alpha3/conversion_test.go

```diff
@@ -38,38 +38,25 @@ func TestFuzzyConversion(t *testing.T) {
 		Scheme:      scheme,
 		Hub:         &v1alpha4.AzureCluster{},
 		Spoke:       &AzureCluster{},
-		FuzzerFuncs: []fuzzer.FuzzerFuncs{overrideImageFuncs, overrideDeprecatedFieldsFuncs},
+		FuzzerFuncs: []fuzzer.FuzzerFuncs{overrideDeprecatedFieldsFuncs},
 	}))
 
 	t.Run("for AzureMachine", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
 		Scheme:      scheme,
 		Hub:         &v1alpha4.AzureMachine{},
 		Spoke:       &AzureMachine{},
-		FuzzerFuncs: []fuzzer.FuzzerFuncs{overrideImageFuncs, overrideDeprecatedFieldsFuncs},
+		FuzzerFuncs: []fuzzer.FuzzerFuncs{overrideDeprecatedFieldsFuncs},
 	}))
 
 	t.Run("for AzureMachineTemplate", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
 		Scheme:      scheme,
 		Hub:         &v1alpha4.AzureMachineTemplate{},
 		Spoke:       &AzureMachineTemplate{},
-		FuzzerFuncs: []fuzzer.FuzzerFuncs{overrideImageFuncs, overrideDeprecatedFieldsFuncs},
+		FuzzerFuncs: []fuzzer.FuzzerFuncs{overrideDeprecatedFieldsFuncs},
 	}))
 
 }
 
-func overrideImageFuncs(codecs runtimeserializer.CodecFactory) []interface{} {
```

Comment:
these were added in v1alpha2 so we should have removed them when moving to alpha4